db/schema.rb

```diff
@@ -52,6 +52,8 @@
     t.boolean  "enable_new_users",                                   default: true
     t.integer  "res_exp_time"
     t.boolean  "enable_guests",                                      default: true
+    t.boolean  "upcoming_checkout_email_active"
```

Comment:
we should probably set this to `default: true` like the `upcoming_checkin_email_active` setting